os-lifecycle-policy.md

```diff
@@ -4,16 +4,18 @@
 
 Each supported operating system has a lifecycle defined by its sponsor organization (for example, Microsoft, Red Hat, Debian, or Apple). The .NET team applies each of those lifecycle schedules to inform adding and removing support for operating system versions.
```

Comment:
I think we need to be crisp on what classifications we're adhering to from each of these platforms.  For example, Debian has three support levels: EOL, EOL LTS, and EOL ELTS.  Which of those do we classify as being not supported for .NET?  That needs to be described for all platforms.

In addition, not all our assets follow this support policy.  For Docker, there is a more restrictive [support policy](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-platforms.md).  Should that be linked to from here?

Reply:
Yes, some definition would be good. The original 2nd paragraph (marked as removed in this or) included additional detail that supported ended when an OS version went out of mainstream or free support.

Reply:
I added a link to the Docker policy.

I added a policy section.